core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/resolvers/dot-container-create.resolver.spec.ts

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do we need to disable this rule? Just in case, if you need any type of a component/response, let us know.